recipes-dotnet/corehost/coreclr_2.0-beta.bb

```diff
@@ -54,7 +53,18 @@ do_install() {
 	install -m 0755 ${src}/mscorlib.dll ${target}
 	install -m 0755 ${src}/System.Private.CoreLib.dll ${target}
 	install -m 0755 ${src}/SOS.NETCore.dll ${target}
+
+	# Create dev package
+	install -d ${D}/opt/dotnet-dev/
+	for i in `ls ${src}/.nuget/pkg/*.nupkg`
+	do
+		install -m 644 ${i} ${D}/opt/dotnet-dev/
```

Comment:
It gets put in a nupkg and then used by the core-host build, doesn't it?

zipinfo packages-split/corefx-dev/opt/dotnet-dev/runtime.ubuntu.16.04-arm.Microsoft.Private.CoreFx.NETCoreApp.4.4.0-beta-25004-01.nupkg
Archive:  packages-split/corefx-dev/opt/dotnet-dev/runtime.ubuntu.16.04-arm.Microsoft.Private.CoreFx.NETCoreApp.4.4.0-beta-25004-01.nupkg
Zip file size: 5842541 bytes, number of entries: 159
?---------  2.0 unx      550 b- defN 17-Feb-24 12:09 _rels/.rels
?---------  2.0 unx     1216 b- defN 17-Feb-24 12:09 runtime.ubuntu.16.04-arm.Microsoft.Private.CoreFx.NETCoreApp.nuspec
?---------  2.0 unx     9323 b- defN 17-Feb-24 12:09 dotnet_library_license.txt
?---------  2.0 unx     1587 b- defN 17-Feb-24 12:09 ThirdPartyNotices.txt
?---------  2.0 unx       41 b- defN 17-Feb-24 12:09 version.txt
?---------  2.0 unx        0 b- stor 17-Feb-24 12:09 useSharedDesignerContext.txt
?---------  2.0 unx   433664 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/Microsoft.CSharp.dll
.........cut....
?---------  2.0 unx    41472 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.IO.FileSystem.DriveInfo.dll
?---------  2.0 unx     6144 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.IO.FileSystem.Primitives.dll
?---------  2.0 unx    40960 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.IO.FileSystem.Watcher.dll
?---------  2.0 unx    37376 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.IO.IsolatedStorage.dll
?---------  2.0 unx    33792 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.IO.MemoryMappedFiles.dll
?---------  2.0 unx    57344 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.IO.Pipes.dll
?---------  2.0 unx     5632 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.IO.UnmanagedMemoryStream.dll
?---------  2.0 unx   124416 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.Linq.dll
?---------  2.0 unx   550912 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.Linq.Expressions.dll
?---------  2.0 unx   208896 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.Linq.Parallel.dll
?---------  2.0 unx    59392 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.Linq.Queryable.dll
?---------  2.0 unx   286208 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.Net.Http.dll
?---------  2.0 unx   139264 b- defN 17-Feb-24 12:09 runtimes/ubuntu.16.04-arm/lib/netcoreapp2.0/System.Net.HttpListener.dll